pkg/runtime/engine/config/config_linux.go

```diff
@@ -0,0 +1,133 @@
+package config
```

Comment:
Missing copyright header here

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.

package config
```

Reply:
thanks!